core/src/test/java/foundation/stack/datamill/http/builder/ElseBuilderTest.java

```diff
@@ -0,0 +1,72 @@
+package foundation.stack.datamill.http.builder;
+
+import foundation.stack.datamill.http.Method;
+import foundation.stack.datamill.http.Response;
+import foundation.stack.datamill.http.Route;
+import foundation.stack.datamill.http.ServerRequest;
+import foundation.stack.datamill.http.impl.RouteBuilderImpl;
+import org.junit.Before;
+import org.junit.Test;
+import rx.Observable;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Israel Colomer (israelcolomer@gmail.com)
+ */
+public class ElseBuilderTest {
```

Comment:
It's a bit weird to have tests for ElseBuilder - the interface is separate from RouteBuilder mostly to have the fluent chaining make sense. These tests should be in RouteBuilderTest